apis/capabilities/v1beta1/policyregistry_types.go

```diff
@@ -32,70 +31,70 @@ import (
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
 const (
-	PolicyKind = "Policy"
+	PolicyRegistryKind = "PolicyRegistry"
 
-	// PolicyInvalidConditionType represents that the combination of configuration
-	// in the PolicySpec is not supported. This is not a transient error, but
+	// PolicyRegistryInvalidConditionType represents that the combination of configuration
+	// in the PolicyRegistrySpec is not supported. This is not a transient error, but
 	// indicates a state that must be fixed before progress can be made.
-	PolicyInvalidConditionType common.ConditionType = "Invalid"
+	PolicyRegistryInvalidConditionType common.ConditionType = "Invalid"
 
-	// PolicyReadyConditionType indicates the policy has been successfully synchronized.
+	// PolicyRegistryReadyConditionType indicates the policy registry has been successfully synchronized.
 	// Steady state
-	PolicyReadyConditionType common.ConditionType = "Ready"
+	PolicyRegistryReadyConditionType common.ConditionType = "Ready"
 
-	// PolicyFailedConditionType indicates that an error occurred during synchronization.
+	// PolicyRegistryFailedConditionType indicates that an error occurred during synchronization.
 	// The operator will retry.
-	PolicyFailedConditionType common.ConditionType = "Failed"
+	PolicyRegistryFailedConditionType common.ConditionType = "Failed"
 )
 
-// PolicySchemaSpec defines the desired Policy schema
-type PolicySchemaSpec struct {
-	// Name is the name of the policy schema
+// PolicyRegistrySchemaSpec defines the desired Policy Registry schema
```

Comment:
In all places `Policy` has been found it has been replaced by `PolicyRegistry`. If there's some place where it should still be called `Policy`because it indicates a real Policy let me know